### PR TITLE
Project Week 10 - Survey

### PR DESCRIPTION
**Pull request by jessika-hage**
Link to my deployed project:

https://hotel-survey.netlify.app/

My reflections on how this week's project turned out:

I'm very satisfied about my project and especially that I managed to do the question counter and the progress bar!

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+The radio buttons styling was also time consuming, and there google helped me.
+I also spent a little time on the progress bar. Googled some but I couldn't really manage to find something to apply in my project (just one piece) so I tried my way out and finally made it to work. That was a very good feeling and maybe a confirmation that I'm really starting to learn and understand React!
+
+Other than the above it went relatively smooth and I think it was a fun project! If I would have had more time I would have made a question that shows a different question depending on the answer. And change the topic to be something more fun :) 
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://hotel-survey.netlify.app/
```
Comment (leelalaim) [inline README.md:15]: Very nice design! :)

Comment (leelalaim) [inline README.md:15]: Very nice design :)

File: code/public/index.html [hunk 2]
```diff
@@ -13,6 +13,8 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans&family=Playfair+Display:wght@500&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/37c653cfef.js" crossorigin="anonymous"></script>
     <title>Technigo React App</title>
```
Comment (leelalaim) [inline code/public/index.html:18]: You can change the project title so it would be project related

File: code/src/components/CountrySelect.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import { Button } from './Button';
+
+// Component for dropdown to select country
```
Comment (leelalaim) [inline code/src/components/CountrySelect.js:5]: Nice that you have comments. It becomes easier to read the code :)

File: code/src/components/QuestionCounter.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+      {question <= 5 && question > 0 ? (
+        <div className="progress-bar">
+          <div className="progress-filler" style={{ width: `${percentage}%` }}>
+          </div>
+        </div>
+      ) : ""}
+      {question <= 5 && question > 0 ? (<>{question} / {total}</>) : ""}
+    </div>
+  )
+}
```
Comment (leelalaim) [inline code/src/components/QuestionCounter.js:17]: Awesome that you have implemented the counter!

**Review by leelalaim — APPROVED** [review]
Nice and neat code, easy to read. Well done!

**Review by KarolCislowski — APPROVED** [review]
Awesome Project!

**Pull request closed without merging**